### ISSUE TITLE
angular-xml not correctly recognizing all RSS feeds as XML

### ISSUE THREAD

**Issue by smromain** [johngeorgewright/angular-xml#30]
Just started using the new version of angular-xml in a podcast application, where response headers are usually returned as ```text/xml;charset=UTF-8```, but occasionally returned as ``` application/rss+xml ``` by certain feeds, such as: http://wtfpod.libsyn.com/rss.

I noticed that in the angular-xml.js file, whether or not a response is XML is determined by the following function:

```javascript 
function responseIsXml(response) {
    var contentType = response.headers('content-type'),
        XML = '/xml',
        minIndex = 'text/xml'.indexOf(XML);
    if (contentType) {
      return contentType.indexOf(XML) >= minIndex;
    } else {
      return false;
    }
  }
```

I've thought about making certain modifications to the functionality such as: 

``` javascript
  function responseIsXml(response) {
    var contentType = response.headers('content-type');
    if (contentType) {
      return contentType.indexOf('xml') > -1;
    } else {
      return false;
    }
  }
```
but I do worry that this may result in some gotchas. Any thoughts?

**Comment by johngeorgewright** [comment]
It has crossed my mind once before. I couldn't decide what route to go down. I'd be happy to do that or maybe even a bit of regex `/(\/|+)xml/`? Would that be overkill?

**Comment by smromain** [comment]
I actually worked on it earlier today. I don't think the regex string is overkill at all - that's how I did it. I forked it and implemented the code I worked on. Will submit pull request, let me know what you think.

**Comment by johngeorgewright** [comment]
Released as v2.0.1. Thanks for contributing!
